model/src/wmgridmd.F90

```diff
@@ -4166,7 +4172,10 @@ SUBROUTINE WMGEQL
                           MAPODI(IY  ,IX+1) .GE. 0. ) ) THEN
                 MASKI(IY,IX) = .TRUE.
 #ifdef W3_O12
-                IF ( IMPROC.EQ.NMPERR ) WRITE (MDSE,1020) I, IX, IY
+                IF ( IMPROC.EQ.NMPERR ) THEN
+                  WRITE (MDSE,1020) I, IX, IY
+                  WRITE (*,*) 'TESTING 4177 ERROR 1020', I, IX, IY
```

Comment:
Remove this line as it seems to be a repeat of the formatted write statement above?

Reply:
Removed!